docs/.vitepress/theme/components/CarbonAds.vue

```diff
@@ -1,9 +1,9 @@
 <template>
-  <div class="carbon-ads" ref="el"></div>
+  <div class="carbon-ads" ref="el" />
 </template>
 
 <script>
-import { onMounted, ref } from 'vue'
+import { ref, onMounted } from 'vue'
```

Comment:
this is not necessary

Reply:
Yep, I figured that it would be better to swap the order, as it makes the imports match the two most common import order conventions: alphabetically and by order of use.  I'll remove it, though.